From 8b5af0694bb82b41d4ae180fac9972029066f90a Mon Sep 17 00:00:00 2001
From: Siddharth <sidsri1502@gmail.com>
Date: Tue, 25 Aug 2026 15:49:22 +0530
Subject: [PATCH] fix(test): update expected format for unrecognised extension
 to yaml (#2837)

When unrecognised extensions are passed, FormatStringFromFilename defaults to 'yaml' (as handled in pkg/yqlib/format.go and tested in pkg/yqlib/format_test.go).

This updates the test expectation in test/format_test.go for 'file.unknown' to 'yaml' so that go test ./test passes.
---
 test/format_test.go | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/test/format_test.go b/test/format_test.go
index ffa41ee1ea..4ad4daeecf 100644
--- a/test/format_test.go
+++ b/test/format_test.go
@@ -17,7 +17,7 @@ func TestFormatStringFromFilename(t *testing.T) {
 		{"FILE.JSON", "json"},
 		{"file.properties", "properties"},
 		{"file.xml", "xml"},
-		{"file.unknown", "unknown"},
+		{"file.unknown", "yaml"},
 
 		// filenames without extensions
 		{"file", "yaml"},

